api/c/indigo/src/indigo_layout.cpp

```diff
@@ -126,6 +128,16 @@ CEXPORT int indigoLayout(int object)
     INDIGO_END(-1);
 }
 
+CEXPORT int indigoLayout(int object)
+{
+    return _indigoLayout(object, false);
+}
+
+CEXPORT int indigoLayoutRespectOld(int object)
```

Comment:
create parameter layout-preserve-existing instead of new function

Reply:
Updated